eve/main.yml

```diff
@@ -16,6 +16,22 @@ models:
       source: tests/artifacts
       urls:
         - "*"
+  - ShellCommand: &fix_curl # Workaround, to be removed
+      name: Install curl
+      command: |-
+        apk --update upgrade
+        apk --update add curl
```

Comment:
We don't need this now with the updated image right?

Reply:
removed